.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
Until it's needed, I'd prefer to use `.eslintrc.json`.

This is mostly a stylistic comment, but I prefer to avoid any potential for logic if I can help it.